olmo/train.py

```diff
@@ -1094,10 +1111,12 @@ def on_trace_ready(p):
                     self.global_train_examples_seen_this_epoch += global_batch_size
                     self.global_train_tokens_seen += global_batch_size * seq_len
                     speed_monitor.batch_start(
-                        self.global_train_tokens_seen,
-                        batch_size * seq_len,  # num tokens in batch for this device
+                        global_total_tokens=self.global_train_tokens_seen,
+                        device_batch_num_tokens=batch_size * seq_len,  # num tokens in batch for this device
                         # We start monitoring speed after the first batch since the first
                         # batch might be an outlier due to compiling and other initialization overhead.
+                        num_fwd_flops=self.model.num_fwd_flops,  # this is per sequence
+                        num_bck_flops=self.model.num_bck_flops,  # this is per sequence
```

Comment:
"this is per sequence" ... it's per-token now, right?

Reply:
changed